packages/mask/src/plugins/Tips/constants.ts

```diff
@@ -0,0 +1 @@
+export const enableRequirement = [{ name: 'EVM Chain', icon: new URL('./assets/evmChains.png', import.meta.url) }]
```

Comment:
Define it in the definition of the `Tips` plugin. Setup the `enableRequirements` property of plugin definition.